xapi_db_load/backends/clickhouse_lake.py

```diff
@@ -5,34 +5,47 @@
 
 
 class XAPILakeClickhouse:
+    client = None
+
     def __init__(self, db_host="localhost", db_port=18123, db_username="default",
                  db_password=None, db_name=None):
         self.host = db_host
         self.port = db_port
         self.username = db_username
         self.database = db_name
+        self.db_password = db_password
 
         self.event_raw_table_name = "xapi_events_all"
         self.event_table_name = "xapi_events_all_parsed"
         self.event_table_name_mv = "xapi_events_all_parsed_mv"
+        self.get_org_function_name = "get_org_from_course_url"
```

Comment:
Need to add this to the XAPILRSRalphClickhouse class too, otherwise running `tutor dev do load-xapi-test-data` with the  current [oars/clickhouse-demo-xapi-data.sh](https://github.com/openedx/tutor-contrib-oars/blob/bmtcril/consolidate_clickhouse/tutoroars/templates/oars/apps/scripts/oars/clickhouse-demo-xapi-data.sh#L9) `--backend ralph_clickhouse` throws:

```
AttributeError: 'XAPILRSRalphClickhouse' object has no attribute 'get_org_function_name'
```

Reply:
Fixed in latest commit